backend/src/Models/CastVoteStore.ts

```diff
@@ -29,11 +29,12 @@ export default class CastVoteStore {
             ballot.date_submitted,
             ballot.ip_address,
             JSON.stringify(ballot.votes),
-            JSON.stringify(ballot.history),
+            JSON.stringify(ballot.history),,
```

Comment:
Is this double comma intentional? Seems wrong to me.

Reply:
Good catch, was a typo